ideScripts/updateTasks.py

```diff
@@ -440,15 +437,14 @@ def getRunTask(self):
 
         buildData = build.BuildData().getBuildData()
         jsonTaskData["label"] = tmpStr.taskName_CPU_run
-        jsonTaskData["command"] = buildData[self.bStr.openOCDPath]
+        jsonTaskData["command"] = buildData[self.bStr.openOcdPath]
         jsonTaskData["args"] = []
-        jsonTaskData["args"].append("-f")
-        jsonTaskData["args"].append(buildData[self.bStr.openOCDInterfacePath])
-        jsonTaskData["args"].append("-f")
-        jsonTaskData["args"].append(buildData[self.bStr.openOCDTargetPath])
+        for arg in buildData[self.bStr.openOcdConfig]:

```

Comment:
As mentioned before, unparsing user configuration string to get files from and than building the same string back again... I would rather stick to the old way.

Reply:
The merit here is that there are cases where users will not want to use two configuration scripts in OpenOCD, so this supports as many as the user would like.